README.md

```diff
@@ -5,85 +5,51 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/mattermost/mattermost-plugin-circleci)](https://goreportcard.com/report/github.com/mattermost/mattermost-plugin-circleci)
 [![Mattermost Community Channel](https://img.shields.io/badge/Mattermost%20Community-~Plugin%3A%20CircleCI-blue)](https://community.mattermost.com/core/channels/plugin-circleci)
 
-The [CircleCI Orb for Mattermost Plugin](https://github.com/nathanaelhoun/circleci-orb-mattermost-plugin-notify) by [@nathanaelhoun](https://github.com/nathanaelhoun) interacts with jobs, builds, or workflows, and receives notifications in Mattermost channels. The Mattermost CircleCI plugin uses a personal API token to connect your Mattermost account to CircleCI to interact with the API.
+**Help Wanted Tickets [here]**
```

Comment:
Should there be a link?

Reply:
I've included the information to align with the provided outline. If you have a specific link you'd like me to add, please provide it, and I'll incorporate it accordingly.

Reply:
Here's the link that should be added, @heisdinesh: ``https://github.com/mattermost/mattermost-plugin-circleci/issues``